src/modules/shared/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,149 @@
+				<div style={column}>
+					<h4>Simple</h4>
+
+					<br />
+
+					<Card {...args} image={undefined} title={undefined}>
+						The simplest card
+					</Card>
+				</div>
+
+				<div style={column}>
+					<h4>Border</h4>
+
+					<br />
+
+					<Card {...args} title="A card with a zinc-colored border">
+						This card is still pretty basic but also has an image and a title
+					</Card>
+				</div>
+
+				<div style={column}>
+					<h4>Edgeless</h4>
+
+					<br />
+
+					<Card
+						{...args}
+						edge="none"
+						padding="vertical"
+						title={<b>A card with no border</b>}
+					>
+						Now that&apos;s more like it, this card has an image, a title in bold, gets
+						rid of the border and adds some much needed vertical padding
+					</Card>
+				</div>
+
+				<div style={column}>
+					<h4>Padded</h4>
+
+					<br />
+
+					<Card
+						{...args}
+						padding="both"
+						title="A card with a few more things"
+						subtitle="Including a subtitle"
+						toolbar="both"
+					>
+						Things keep getting better; Now we have a card that has padding on both the
+						content and image, reintroducs the border, adds a subtitle and adds
+						&quot;both&quot; in the toolbar-area
+					</Card>
+				</div>
+
+				<div style={column}>
+					<h4>Wide</h4>
+
+					<br />
+
+					<Card
+						{...args}
+						padding="content"
+						title="A card with padding on just the content"
+						toolbar="content"
+					>
+						Okay, maybe we&apos;d like to give our visual just that little bit of extra
+						space...
+					</Card>
+				</div>
+			</section>
+
+			<br />
+
+			<section style={{ ...container }}>
+				<div style={column}>
+					<h4>Padded + Horizontal</h4>
+
+					<br />
+
+					<Card
+						{...args}
+						padding="both"
+						orientation="horizontal"
+						title={<b>A horizontal card with padding on both image and content</b>}
+						subtitle="Studio Hyperdrive (15 dec. 2022) "
+						toolbar="both + horizontal"
+					>
+						<TruncateMarkup lines={2}>
+							<span>
+								Aenean nec feugiat nisi. Pellentesque vel nunc sit amet augue
+								tincidunt egestas. Cras vitae molestie leo. Nullam sed arcu aliquet,
+								porta massa vel, feugiat sapien.
+							</span>
+						</TruncateMarkup>
+					</Card>
+				</div>
+				<div style={column}>
+					<h4>Wide + Horizontal</h4>
+
+					<br />
+
+					<Card
+						{...args}
+						padding="content"
+						orientation="horizontal"
+						title="A horizontal card with padding on just the content"
+						toolbar="content + horizontal"
+					>
+						<TruncateMarkup lines={2}>
+							<span>
+								This card is <b>limited</b> to two lines of text in the description,
+								anything that goes beyond that is cut off and replaced with an
+								ellipsis.
+							</span>
+						</TruncateMarkup>
+					</Card>
+				</div>
+			</section>
+		</>
+	);
+};
+
+export const Overview: ComponentStory<typeof Card> = Template.bind({});
```

Comment:
Je kan ipv een overview met alle varianten meerdere instances maken die dan telkens onder card komen in Storybook.
```
export const Simple: ComponentStory<typeof Card> = Template.bind({});
Simple.args = {
	title={undefined}
        image={undefined}
};

...